dashboard/app/controllers/activities_controller.rb

```diff
@@ -17,7 +17,7 @@ class ActivitiesController < ApplicationController
   MIN_LINES_OF_CODE = 0
   MAX_LINES_OF_CODE = 1000
 
-  use_reader_connection_for_route(:milestone)
+  use_reader_connection_for_route(:milestone, gatekeeper_controlled: true)
```

Comment:
One question: I gated the new functionality behind two flags: the gatekeeper flag, obviously, but also this optional parameter to the `use_reader_connection_for_route` helper method. I did the second one just to save on excess `Gatekeeper.allows` calls, but after taking a second look at the way gatekeeper stores its values internally I realized these calls are much cheaper than I thought.

So, now I'm thinking I might remove this param and apply this change to every existing read split route, not just the ones that we explicitly add this parameter to. Thoughts?

Reply:
I think it makes sense to make it apply to all uses of `use_reader_connection_for_route`, that makes the flag a little more intuitive and potentially useful down the line.